AFQ/tests/test_api.py

```diff
@@ -192,7 +192,7 @@ def test_AFQ_data_waypoint():
         'sub-01_sess-01_dwi_space-RASMM_model-DTI_desc-det-AFQ-clean_tractography_idx.json'))  # noqa
 
     tract_profiles = pd.read_csv(myafq.tract_profiles[0])
-    assert tract_profiles.shape == (800, 5)
+    assert tract_profiles.shape == (400, 5)
```

Comment:
Hmm. Why are there only half the numbers here?

Reply:
Two columns were removed: scalar and profile. scalar was the label (ie "dti_fa") and profile was the number. Now, instead, there are two columns of numbers: "dti_md", "dti_fa". This means it is only half the length. For each node, there is only one row instead of one for each scalar.

Reply:
Gotcha! That makes sense.